### PR TITLE
Blank State - Add Open in Shell

### LINKED ISSUE #6674: Add option - [show in git shell]
In the latest update, I noticed GitHub Desktop now has a variety of convenience buttons. I have found myself frequently using the [show in explorer] function. I was wondering if it is possible to add a similar button [show in git shell] that opens a command prompt in the directory of the repository.

suggestion

### PR DESCRIPTION
**Pull request by say25**
## Overview
**Closes #6674**

## Description

- Adds open in Shell to Blank State

## Release notes

Notes: Adds open in Shell to Blank State

## Screenshots
**Before**
before

**After**
after

### CODE REVIEW

File: app/src/ui/changes/no-changes.tsx [hunk 1]
```diff
@@ -318,33 +318,43 @@ export class NoChanges extends React.Component<
...
+      'open-external-editor',
+      'external editor'
+    )
+  }
+
+  private renderOpenInShell() {
+    return this.tryRenderMenuBackedAction('open-in-shell', 'shell')
+  }
+
+  private tryRenderMenuBackedAction(menuId: MenuIDs, whatToOpen: string) {
```
Comment (say25) [inline app/src/ui/changes/no-changes.tsx]: I dont love `whatToOpen` but I couldn't determine come up with a better variable name at the moment.

Comment (j-f1) [reply app/src/ui/changes/no-changes.tsx]: `programType`?

**Comment by shiftkey** [comment]
Tagging this as `more-information-needed` because of the discussion in #6674

**Comment by shiftkey** [comment]
Closing this out until we agree on whether this is the right change in #6674

**Pull request closed without merging**